tests/pages/services/ServiceTable-cy.js

```diff
@@ -234,9 +234,9 @@ describe("Service Table", function() {
         .should("to.have.length", 1);
 
       cy
-        .get(".modal pre")
+        .get(".modal-body")
```

Comment:
I would just remove this test entirely because it's duplicated, but not bug deal for me to block from merging.